src/reversible_tokenize.jl

```diff
@@ -1,5 +1,23 @@
 # A simple reversible tokenizer
+"""
+A simple reversible tokenizer
 
+```
+tokenized = rev_tokenizer(instring)
+de_tokenized = rev_detokenizer(token)
+```
+
+The rev_tokenize tokenizer splits into token based on space and punctuations and in 
+addition it leaves some MERGESYMBOLS for the tokens to be re-arranged when needed 
+using the rev_detokenize.
+It uses a character based approach for splitting and re-merging.
+
+Parameters:
+
+- instring		= Input string to be tokenized 
+- token			= Collection to tokens i.e String Array
+
+"""
 MERGESYMBOL = '~'
```

Comment:
Use a character from unicode private use area. e.g. `\ue302`

Reply:
I was trying to do that for a while... But this error persists if I use any of the `\` symbols for `MERGESYMBOL` , thus I added `~`
```
Some of 1 ↹,↹ 0000 households (↹ usually ↹, a minority ↹) ate breakfast ↹.
StringIndexError("Some of 1 ↹,↹ 0000 households (↹ usually ↹, a minority ↹) ate breakfast ↹.", 12)

Stacktrace:
 [1] string_index_err(::String, ::Int64) at ./strings/string.jl:12
 [2] getindex_continued(::String, ::Int64, ::UInt32) at ./strings/string.jl:218
 [3] rev_detokenize(::Array{SubString{String},1}) at ./strings/string.jl:211
 [4] top-level scope at In[21]:1

```